lang/janet/Portfile

```diff
@@ -5,8 +5,8 @@ PortGroup           github 1.0
 PortGroup           legacysupport 1.1
 PortGroup           makefile 1.0
 
-github.setup        janet-lang janet 1.15.5 v
-revision            2
+github.setup        janet-lang janet 1.22.0 v
+revision            3
```

Comment:
the revision field should be reset to zero when updating the version

Reply:
Cheers, will fix

Reply:
All good now @reneeotten ?